### DISCUSSION TITLE
Rilascio 1.29:  dati AIRE

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#911] [Supporto servizi cooperativi] [answered]
Con il rilascio 1.29 viene introdotta la gestione dei soggetti AIRE; in particolare, su "Model Soggetto" si trova ora la sezione 

![image](https://github.com/italia/ansc/assets/26415430/8c45811e-693e-49d2-8b89-d01c3db5bf0c)

che dovrebbe essere valorizzato nel caso in cui l'ulteriore nuovo campo:

![image](https://github.com/italia/ansc/assets/26415430/fa00a1cb-5bcd-4d40-8d85-d88c2636f979)

assuma valore "true".
 
Abbiamo però rilevato che quest'ultimo nuovo campo:

![image](https://github.com/italia/ansc/assets/26415430/fa00a1cb-5bcd-4d40-8d85-d88c2636f979) 

non è presente per i titolari in alcuni casi d'uso; 
esempio: non troviamo il campo in questione in
"Matr_001" (ma ci pare in effetti per tutti gli atti di matrimonio)
"Morte" (ma ci pare in effetti per tutti gli atti di morte)

mentre lo troviamo nei casi d'uso di nascita.

Si tratta di un errore nella documentazione?

**Comment by mttfranci** [comment, marked as answer]
Salve, Mentre i campi sul comune di idscrizione AIRE sono presenti per tutti i registri. il campo flagTrasmissioneResidenzaEstera serve solo per la nascita, per dare eventualmente indicazione, del genitore da cui il bambino eredita la residenza (ai fini delle notifiche).

Saluti 

Assistenza ANSC

PS: il campo è comunque opzionale ed è possibile inviare il comune di iscrizione AIRE direttamente come attributo dell'intestatario.

**Comment by antonioluberto** [comment]
Qualcosa non è chiaro: come fa un booleano (che per definizione può assumere solo i due valori: "true" o "false") ad indicare se il nato eredita il Comune di residenza dal padre o dalla madre?

**Reply by mttfranci** [reply]
Salve, è booleano è sufficiente perchè va indicato su uno dei genitori.

Saluti 

Assistenza ANSC
